easysid.cpp

```diff
@@ -26,7 +37,13 @@ EXPORT void start_capture(const char *fn)
         delete wav;
         wav = NULL;
     }
-    wav = makePcmOutputWav(fn);
+    if (pcm == NULL) {
+        if (not initAudioOutput()) {
+            // TODO: Error condition
+            return;
+        }
+    }
+    wav = makePcmOutputWav(fn, pcm->getSampleRate());
```

Comment:
What happened to the global sample rate? Surely it makes more sense to hold the *actual* sample rate in a global, since it applies to the entire audio pipeline. It does not make sense to have the actual sample rate stored inside an object that you have to create just to get the current value out of.